src/uct/api/uct_def.h

```diff
@@ -235,4 +238,77 @@ typedef size_t (*uct_pack_callback_t)(void *dest, void *arg);
 typedef void (*uct_unpack_callback_t)(void *arg, const void *data, size_t length);
 
 
+/**
+ * @ingroup UCT_TAG
+ * @brief Callback to process unexpected eager tagged message.
+ *
+ * This callback is invoked when tagged message sent by eager protocol is
+ * arrived and there is no corresponding tag posted.
```

Comment:
Minor grammar cleanup:
  This callback is invoked when a tagged message sent by the eager protocol has
  arrived and no corresponding tag has been posted.